src/destructors.md

```diff
@@ -1,11 +1,15 @@
 # Destructors
+r[destructors]
 
+
+r[destructors.general]
```

Comment:
```suggestion
r[destructors.intro]
```

In the other chapter, we went with `.intro` for cases like these.  The particular name probably doesn't matter, but we should be consistent.  `intro` is shorter, so maybe let's do that.